pkg/appmanager/appManager.go

```diff
@@ -1457,6 +1471,7 @@ func (appMgr *Manager) handleConfigForType(
 
 	deactivated := false
 	if _, ok := svcPortMap[pool.ServicePort]; !ok {
+			pool.ServiceName, svcKey.Namespace)
```

Comment:
This feels out of place.

Reply:
It is.  It was part of a print statement I forgot to completely remove.